src/css/_styles.scss

```diff
@@ -0,0 +1,161 @@
+@import 'reset';
```

Comment:
If you are having issues with the partials you may want to do your import into the base.scss, that seemed to fix error I was having